src/TestProblem/Hydro/CCSN/Record_CCSN.cpp

```diff
@@ -156,16 +159,16 @@ void Record_CCSN_CentralQuant()
 
       for (int v=0; v<NCOMP_TOTAL; v++)   u[v] = amr->patch[ amr->FluSg[lv] ][lv][PID]->fluid[v][k][j][i];
 
-
-      FILE *file_cent_quant = fopen( filename_central_quant, "a" );
 #     ifdef YE
       const real Ye = u[YE] / u[DENS];
 #     else
       const real Ye = (real)0.0;
 #     endif
-      fprintf( file_cent_quant, "%15.7e %12ld %16.7e %16.7e %16.7e %16.7e %16.7e\n",
+
```

Comment:
Maybe we need to add one empty line back?

Reply:
Which empty line was removed here?